src/amd/compiler/meson.build

```diff
@@ -93,7 +93,7 @@ libaco = static_library(
     idep_nir_headers, idep_amdgfxregs_h,
   ],
   c_args : [c_vis_args],
-  cpp_args : [cpp_vis_args],
+  cpp_args : [cpp_vis_args, '-std=c++14'],
```

Comment:
You shouldn't have to do this

Recreating your build directory or running `meson configure <build dir> -Dcpp_std=c++14` should fix this

Reply:
Yeah, I will try that. My meson is a bit rusty so this was the dirty workaround